admin/netdata/Makefile

```diff
@@ -88,7 +91,6 @@ define Package/netdata/install
 	rm $(1)/usr/share/netdata/web/images/*.png
 	rm $(1)/usr/share/netdata/web/images/*.gif
 	rm $(1)/usr/share/netdata/web/images/*.ico
-	rm -rf $(1)/usr/share/netdata/web/old
```

Comment:
Why?

Reply:
My submission is modified from version 1.2.0
If you feel that some paragraphs are inappropriate, you can delete it

Reply:
Well, I have completed the modification according to your prompt